CHANGELOG.md

```diff
@@ -1,11 +1,15 @@
 # Changelog
 
 ## 0.8.2
+### Changed
+- Model deserialization errors are now and `out` variable not a list you pass in.
```

Comment:
I suggest: The errors parameter for Model.FromJson now has the `out` modifier. It no longer takes a default value.